test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso

```diff
@@ -180,6 +180,19 @@ add_column_operation_specs suite_builder setup =
             3.1 . round 16 . should_fail_with Illegal_Argument
             3.1 . round -16 . should_fail_with Illegal_Argument
 
+        group_builder.specify "Non-builtin integer column rounding" <|
+            cases =
+                [[0, 0, 0], [3, 0, 3], [-3, 0, -3], [3, 1, 3], [-3, 1, -3]] +
```

Comment:
Do + work at the end for multi-line?

Reply:
Ah, I knew that was going to bite me someday. Fixed.